runbotics/runbotics-scheduler/src/auth/auth.module.ts

```diff
@@ -42,7 +44,7 @@ const GlobalFeatureKeyGuard = {
             })
         }),
     ],
-    providers: [AuthService, JwtStrategy, GlobalRoleGuard, GlobalFeatureKeyGuard],
-    exports: [AuthService, PassportModule],
+    providers: [AuthService, JwtStrategy, GlobalRoleGuard, GlobalFeatureKeyGuard, MicrosoftSession],
+    exports: [AuthService, PassportModule, MicrosoftSession],
```

Comment:
fyi, you don't have to export it if it is used only in the same module in which you use it (analyze your case)